File: LeetCode/Javascript/shuffleTheArray.js
var shuffle = function(nums, n) {
    let resul = []
    for (let i=0; i<n; i++){
        resul.push(nums[i]);
        resul.push(nums[i+n]);
    }
    return resul;
};

let nums = prompt("nums")
nums = nums.substring(1,nums.length-1).split(",").map(tempVar => parseInt(tempVar,10));
let n = parseInt(prompt("n"),10);

console.log(shuffle(nums,n));

File: LeetCode/Python/shuffleTheArray.py
# https://leetcode.com/problems/shuffle-the-array/
# Reprak11
from typing import List

class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        salida=[]
        for i in range (0,n):
            salida.append(nums[i])
            salida.append(nums[i+n])
        return salida


if __name__ == "__main__":
    nums = input()[1:-1]
    nums = list(map(int,nums.split(",")))
    n = int(input())
    print(Solution.shuffle(Solution,nums,n))